numpy/lib/tests/test_format.py

```diff
@@ -892,10 +895,12 @@ def test_large_archive(tmpdir):
     with open(fname, "wb") as f:
         np.savez(f, arr=a)
 
+    del a
```

Comment:
I took the liberty to add this, but I am not quite sure that PyPy will clean up quickly enough on a `del` here?  So on PyPy, should we use `2 * (2 * 2**30)` bytes for the `requires_memory`?